app/src/main/java/com/concordia/mcga/fragments/TransportButtonFragment.java

```diff
@@ -59,41 +71,49 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         publicTransportFAB.setOnClickListener(this);
         shuttleFAB.setOnClickListener(this);
 
-        //Some test values
-        setWalkTime(1, 30);
-        setBikeTime(1, 0);
-        setCarTime(0, 30);
-        setPublicTransportTime(0, 45);
-        setShuttleTime(0, 30);
+        //Set initial transport type and icon
+        this.transportType = Transportation.PUBLIC_TRANSPORT;
+        swapIcons(Transportation.PUBLIC_TRANSPORT.getIconID());
 
         return view;
     }
 
     @Override
     public void onClick(View clickedView) {
+        toggle();
         switch (clickedView.getId()) {
```

Comment:
There is a code smell here, we're missing the default case. 
Log.e("Fragment","ERROR clicked on a non-existing button");

This is obviously never supposed to happen, but it could if someone added an extra button or click listener...